modules/database/sqlite/db.copy

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Plugin Name: Performance Lab - SQLite module.
+ * Plugin URI: https://github.com/WordPress/performance
+ * Description: Performance plugin from the WordPress Performance Team, which is a collection of standalone performance modules.
+ * Requires at least: 6.0
+ * Requires PHP: 5.6
+ * Version: 1.7.0
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ * Text Domain: performance-lab
+ *
+ * This file is auto-generated and copied from the performance-lab plugin.
+ * Please don't edit this file directly.
+ *
+ * @package performance-lab
+ */
+
+// Bail early if the SQLite implementation was not located in the performance-lab plugin.
+if ( ! file_exists( '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php' ) ) {
+	return;
+}
+
```

Comment:
Can we set some unconditional constant here like `PERFLAB_SQLITE_DB_DROPIN_VERSION`? This would have two benefits:
* Check at any point whether this drop-in file has actually loaded (checking whether `db.php` exists is not the same thing, as it may be another `db.php` drop-in)
* In the future potentially use it to check against the version of the drop-in, to make sure we replace it with a newer version if changes have been made

The version could be set to the same version the drop-in is set to in the header (`1.8.0` probably).

Reply:
Makes sense...
I added a `PERFLAB_SQLITE_DB_DROPIN_VERSION` constant, and performed some additional tweaks required to properly implement it (more detailed error messages, avoid overwriting pre-existing `db.php` files, let the user know if they can't activate the module because there's another drop-in present and so on)